lib/hashes.md

```diff
@@ -5,40 +5,46 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A hash is defined by curly brackets {} and can be viewed as a form of an array
+except the order or list of values are not important with hashes as it is with arrays.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+When calling for a value within a hash, a key must be used instead of an integer
+(unless an integer was used as a key).
```

Comment:
I was thinking of comparing hashes to arrays when I wrote that response. I didn't think it would have been appropriate by saying hashes are like arrays and strings and fixnums are single values not stored in a cell.